tools/mmp/driver.cs

```diff
@@ -284,6 +284,7 @@ static void Main2 (string [] args)
 				{ "tls-provider=", "Specify the default TLS provider", v => { tls_provider = v; }},
 				{ "http-message-handler=", "Specify the default HTTP Message Handler", v => { http_message_provider = v; }},
 				{ "extension", "Specifies an app extension", v => is_extension = true },
+				{ "allow_unsafe_gac_resolution", "Allow msbuild to resolve from the System GAC", v => {} , true }, // Used in Xamarin.Mac.XM45.targets and must be ignored here. Hidden since it is a total hack. If you can use it, you don't need support
```

Comment:
`allow-unsafe-gac-resolution` would be more inline with other options

`msbuild` → `MSBuild`